integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DomainUtils.java

```diff
@@ -139,4 +225,507 @@ public static void deleteDomainResource(String domainNS, String domainUid) {
+                                                 String wlSecretName,
+                                                 String pvName,
+                                                 String pvcName,
+                                                 String clusterName,
+                                                 int replicaCount) {
+    // create the domain custom resource
+    getLogger().info("Creating domain custom resource");
+    Domain domain = new Domain()
+        .apiVersion(DOMAIN_API_VERSION)
+        .kind("Domain")
+        .metadata(new V1ObjectMeta()
+            .name(domainUid)
+            .namespace(domainNamespace))
+        .spec(new DomainSpec()
+            .domainUid(domainUid)
+            .domainHome("/u01/shared/domains/" + domainUid)
+            .domainHomeSourceType("PersistentVolume")
+            .image(WEBLOGIC_IMAGE_TO_USE_IN_SPEC)
+            .imagePullPolicy("IfNotPresent")
+            .imagePullSecrets(Collections.singletonList(
+                new V1LocalObjectReference()
+                    .name(BASE_IMAGES_REPO_SECRET)))
+            .webLogicCredentialsSecret(new V1SecretReference()
+                .name(wlSecretName)
+                .namespace(domainNamespace))
+            .includeServerOutInPodLog(true)
+            .logHomeEnabled(Boolean.TRUE)
+            .logHome("/u01/shared/logs/" + domainUid)
+            .dataHome("")
+            .serverStartPolicy("IF_NEEDED")
+            .serverPod(new ServerPod()
+                .addEnvItem(new V1EnvVar()
+                    .name("JAVA_OPTIONS")
+                    .value("-Dweblogic.StdoutDebugEnabled=false"))
+                .addEnvItem(new V1EnvVar()
+                    .name("USER_MEM_ARGS")
+                    .value("-Djava.security.egd=file:/dev/./urandom "))
+                .addVolumesItem(new V1Volume()
+                    .name(pvName)
+                    .persistentVolumeClaim(new V1PersistentVolumeClaimVolumeSource()
+                        .claimName(pvcName)))
+                .addVolumeMountsItem(new V1VolumeMount()
+                    .mountPath("/u01/shared")
+                    .name(pvName)))
+            .adminServer(new AdminServer()
+                .serverStartState("RUNNING")
+                .adminService(new AdminService()
+                    .addChannelsItem(new Channel()
+                        .channelName("default")
+                        .nodePort(getNextFreePort()))))
+            .addClustersItem(new Cluster() //cluster
+                .clusterName(clusterName)
+                .replicas(replicaCount)
+                .serverStartState("RUNNING")));
+    setPodAntiAffinity(domain);
+
+    // Verify the domain custom resource is created.
+    // Also verify the admin server pod and managed server pods are up and running.
+    createDomainAndVerify(domainUid, domain, domainNamespace, domainUid + "-" + ADMIN_SERVER_NAME_BASE,
```

Comment:
remove this from here and call it from createDomainOnPvUsingWdt instead

Reply:
fixed